### PR TITLE
extended patch for pr 2111

### PR DESCRIPTION
**Pull request by sanidhyamangal**
* [x] Have you followed the guidelines in our
      [Contributing document](https://dvc.org/doc/user-guide/contributing)?

* [x] Does your PR affect documented changes or does it add new functionality
      that should be documented? If yes, have you created a PR for
      [dvc.org](https://github.com/iterative/dvc.org) documenting it or at
      least opened an issue for it? If so, please add a link to it.

-----

This patch updates the content of the shared output data sync.

### CODE REVIEW

File: dvc/command/data_sync.py [hunk 1]
```diff
@@ -116,7 +116,7 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help="Limit the scope to these DVC-files. With -R a directory to search DVC-files in can be specified.",
```
Comment (shcheklein) [inline dvc/command/data_sync.py]: please, check the contributing guide and make sure that you run `black` manually or via the pre-commit hook to enforce the style guide.

**Comment by sanidhyamangal** [comment]
@shcheklein, All the problems are resolved now kindly review this change.

File: dvc/command/data_sync.py [hunk 2]
```diff
@@ -116,7 +116,8 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help="""Limit the scope to these DVC-files.
+        With -R a directory to search DVC-files in can be specified.""",
```
Comment (shcheklein) [inline dvc/command/data_sync.py]: this way it probably means that this line now includes the newline. It's probably not what we want. Have you run the pre-commit hook on the previous patch?

Comment (sanidhyamangal) [reply dvc/command/data_sync.py]: @shcheklein, I tried it every possible way. it is the only way to add both these statements in the help or else the build in Travis will fail as adding these two statements will exceed char limit that is 79 and in this 109. Will it be okay if I change the language of this message?

Comment (shcheklein) [reply dvc/command/data_sync.py]: have you tried to install and run the pre-commit back hook like described here https://dvc.org/doc/user-guide/contributing#development-environment ? It takes care automatically and split the long string properly for you. I highly recommend you to setup the proper dev environment. Alternatively, take a look a the dvc gc command help - there are some long help messages examples.

Comment (sanidhyamangal) [reply dvc/command/data_sync.py]: @shcheklein, Yes I tried them still they were of no use. Check out this screenshot.
Screenshot (20)

Comment (efiop) [reply dvc/command/data_sync.py]: @sanidhyamangal pre-commit hook won't split string lines for you, you need to do that yourself. As @shcheklein suggested, take a look at gc, where it has long help messages and format this one in a similar way.

Comment (sanidhyamangal) [reply dvc/command/data_sync.py]: @efiop  I have updated the requested changes please review them.

File: dvc/command/data_sync.py [hunk 3]
```diff
@@ -116,7 +116,10 @@ def shared_parent_parser():
         "targets",
         nargs="*",
         default=None,
-        help="Limit the scope to these stage files.",
+        help=(
+            "Limit the scope to these DVC-files."
+            " With -R a directory to search DVC-files in can be specified."
```
Comment (jorgeorpinel) [inline dvc/command/data_sync.py]: This affects documentation changes which weren't sent to the dvc.org repo. I'll fix it, but just a note. Thanks

**Pull request merged**